src/DataFramesMeta.jl

```diff
@@ -112,11 +113,15 @@ function fun_to_vec(kw::Expr; nolhs = false)
                 ($(Expr(:tuple, values(membernames)...)) -> $body)
             end
          else
-            output = kw.args[1]
+            if kw.args[1] isa Symbol
+                output = QuoteNode(kw.args[1])
+            elseif onearg(kw.args[1], :cols)
+                output = kw.args[1].args[2]
+            end
```

Comment:
Some comments here about what each branch is handling would be nice.

Reply:
added, thanks!